pkg/restapi/vc/operation/operations.go

```diff
@@ -111,11 +158,65 @@ func createCredential(data *CreateCrendential) (*verifiable.Credential, error) {
 	return validatedCred, nil
 }
 
+func (c *Operation) createProfile(pr *ProfileRequest) (*ProfileResponse, error) {
+	if pr.DID == "" && pr.URI == "" {
```

Comment:
if pr.DID is blank but pr.URI isn't, is any error thrown? If not, you'll probably want to catch that.